data/influx/influx.go

```diff
@@ -38,16 +38,24 @@ func (s *Influx) query(query string, database string) client.Query {
 }
 
 // Query executes the provided query string and returns the results as a JSON object
```

Comment:
Actually, I intend to use helpers as you suggest, for the transformation, as the JSON is a bit nested and would not be very friendly for use in a proto message. I left the JSON because I thought it was useful for testing and debugging.